src/plugins/expression/expression-tile.tsx

```diff
@@ -51,10 +65,23 @@ export const ExpressionToolComponent: React.FC<ITileProps> = observer((props) =>
   }, [content, readOnly]);
 
   const handleChange = (e: FormEvent<MathfieldElementAttributes>) => {
+    const mfLatex = (e.target as MathfieldElement).value;
+    const replacedLatex = replaceLatex(mfLatex);
     trackedCursorPos.current =  mf.current?.position || 0;
-    content.setLatexStr((e.target as any).value);
+    if (mf.current?.value){
+      mf.current.value = replacedLatex;
+    }
+    if (mf.current?.position){
```

Comment:
When the position is 0 this will not be triggered. I'm not sure there any cases where that matters but just to be safe I'd recommend changing this to:
```suggestion
    if (mf.current && trackedCursorPos.current != null){
```

Reply:
Fixed.